perceval/backends/core/meetup.py

```diff
@@ -88,10 +91,18 @@ def __init__(self, group, api_token, max_items=MAX_ITEMS,
         self.group = group
         self.max_items = max_items
         self.api_token = api_token
+        self.is_oauth_token = is_oauth_token
         self.sleep_for_rate = sleep_for_rate
         self.min_rate_to_sleep = min_rate_to_sleep
         self.sleep_time = sleep_time
 
+        if not self.is_oauth_token:
+            warnings.warn(
+                "The use of API keys won't be longer supported, "
+                "use Oauth tokens instead",
```

Comment:
Maybe this message is more clear: `API keys are no longer actively supported by Meetup. Use OAuth2 tokens instead`. It's like the message they have in their page.